src/main/scala/io/iohk/ethereum/rlp/package.scala

```diff
@@ -43,4 +44,11 @@ package object rlp {
     * @throws RLPException if there is any error
     */
   def nextElementIndex(data: Array[Byte], pos: Int): Int = RLP.getItemBounds(data, pos).end + 1
+
+  trait RLPSerializable {
+    def toRLPEncodable: RLPEncodeable
+    def toBytes(implicit di: DummyImplicit): ByteString = ByteString(toBytes: Array[Byte])
+    def toBytes: Array[Byte] = encode(this.toRLPEncodable)
```

Comment:
should this trait extend/share toBytes method with `MessageSerializable` in
https://github.com/input-output-hk/etc-client/pull/186/files#diff-07c07211a1c9ffb837c7b3895bd5321bR14